internal/pkg/deploy/cloudformation/pipeline_test.go

```diff
@@ -87,10 +87,14 @@ func TestCloudFormation_CreatePipeline(t *testing.T) {
 		Stages:          nil,
 		ArtifactBuckets: nil,
 	}
+	mockS3BucketName := "BitterBucket"
```

Comment:
!

Reply:
a la Bitbucket, not a bucket full of bitterness!